### ISSUE TITLE
can this whole model be pretrained?

### ISSUE THREAD

**Issue by Simon-Beck12** [Curtis-Wu/Equivariant-Graph-Transformer#1]
im trying to do molecular property prediction(CYP3A4 Inhibition prediction), 
and i need to pretrain to big datasets.

any suggestion on how to pretrain?

**Comment by Curtis-Wu** [comment]
You can pretrain on large labeled datasets using a standard supervised learning setup (e.g., predicting molecular properties), and then fine-tune the model on your downstream task.

For this project, the model was initialized with pretrained weights from Y. Wang et al., Denoise Pre-training on Non-equilibrium Molecules for Accurate and Transferable Neural Potentials, where the pretraining was done in an unsupervised denoising fashion.

**Comment by Simon-Beck12** [comment]
@Curtis-Wu would it be possible to self supervised pretrain? cause i dont really have much label for cyp3a4 inhibition

**Comment by Curtis-Wu** [comment]
The provided trained model(s) already uses pre-trained weights from Y. Wang et al., Denoise Pre-training on Non-equilibrium Molecules for Accurate and Transferable Neural Potentials. 
Unfortunately it wouldn't be possible to pre-train with the code provided in this repository, please refer to [[Paper]](https://arxiv.org/abs/2303.02216) [[GitHub]](https://github.com/yuyangw/Denoise-Pretrain-ML-Potential).
